src/main/java/org/opentripplanner/gtfs/mapping/LinkStopsAndParentStationsTogether.java

```diff
@@ -27,18 +28,25 @@
     LinkStopsAndParentStationsTogether(
```

Comment:
Add Javadoc explaining that this sets the parent station field, adds child stops to stations, and inherits coordinates as needed from parent stations. Change the name to be a noun instead of a verb (since it's a class). Seeing that this also implies a common interface, it may be a good idea to create this interface with a name like StationElement (rather than emphasizing that it is "StopLike"). This interface would have level information and parent station information.